c2corg_ui/templates/document/diff.html

```diff
@@ -20,7 +20,7 @@ <h2 class="text-center">${title}</h2><br>
         <a href="${request.route_url(
               module + '_archive', id=document_id, lang=lang,
               version=version1['version_id'])}"><span translate>Revision as of</span> {{'${1000 * version1['written_at']}' | date:'medium'}}</a><br/>
-        <span translate>by</span> <a href="#TODO profile for ${version1['user_id']}">${version1['name']}</a>
+        <span translate>by</span> <a href="${request.route_url('profiles_view_id', id=version1['user_id'])}">${version1['name']}</a>
```

Comment:
Would it make sense to have the user locale here? Like in ` c2corg_ui/static/partials/feed.html `?